docs/current/sequence_diagram.mmd

```diff
@@ -0,0 +1,17 @@
+sequenceDiagram
```

Comment:
I didn't "reference"-incorporate this into `2_salesforce-mobile-web-mcp-server.md`, as I don't think there's a way, but I wanted to maintain the diagram outside of the inline representation in the doc. If we need to make updates, we'll need to remember to do it in both places.